cmd/virtual-kubelet/commands/root/root.go

```diff
@@ -102,8 +106,15 @@ func runRootCommand(ctx context.Context, c Opts) error {
 	configMapInformer := scmInformerFactory.Core().V1().ConfigMaps()
 	serviceInformer := scmInformerFactory.Core().V1().Services()
 
-	go podInformerFactory.Start(ctx.Done())
-	go scmInformerFactory.Start(ctx.Done())
+	wg.Add(2)
+	go func() {
+		defer wg.Done()
+		podInformerFactory.Start(ctx.Done())
+	}()
+	go func() {
```

Comment:
Why not use errgroup?

Reply:
I haven't heard of it before, can you elaborate? :)
Will it make any difference?